engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -278,4 +280,37 @@ public static synchronized void changeLoggersLevel(@Nonnull Map<String, String>
     }
     return ImmutableMap.copyOf(matchedClasses);
   }
+
+  /**
+   * Returns logger log file name
+   * @return logFileName
+   */
+  public static String getLogFileName()
+  {
+    Enumeration e = LogManager.getRootLogger().getAllAppenders();
+    while (e.hasMoreElements()) {
+      Appender appender = (Appender)e.nextElement();
+      if (appender instanceof FileAppender) {
+        return ((FileAppender)appender).getFile();
```

Comment:
We have hardcoded "RFA" in our code (check classes StramClient and LaunchContainerRunnable), is it a good idea to look for "RFA" specifically? If RFA isn't found we should return "null".